### PR TITLE
Chessx 1.2.2 (new cask)

### PR DESCRIPTION
**Pull request by gmunkhbaatarmn**
New Cask: "ChessX".
http://chessx.sourceforge.net/

ChessX screenshot

### CODE REVIEW

File: Casks/chessx.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'chessx' do
+  version '1.2.2'
+  sha256 '2a2064eed320867b6e054ddfee1e0f4ea0c307e3648c28835f0909568ce7b74f'
+  url "http://downloads.sourceforge.net/project/chessx/chessx/#{version}/chessx-#{version}.dmg"
+
+  name 'ChessX'
+  homepage 'http://chessx.sourceforge.net/'
+  license :gpl
+
+  pkg 'chessx-installer.mpkg'
```
Comment (alebcay) [inline Casks/chessx.rb]: Can you write an `uninstall` stanza for this package? Do you need any help with that?

Comment (gmunkhbaatarmn) [inline Casks/chessx.rb]: Now, `uninstall` stanza added. See next line on cask.
But it's my first time on cask adding. I've attached some info for explain my understanding of `:pkgutil` key:

```
# Before install
$ pkgutil --packages | grep 'chessx'
<empty result>

# After install & uninstall
$ pkgutil --packages | grep 'chessx'
net.sourceforge.chessx
```

**Comment by phinze** [comment]
Uninstall stanza looks good - can you squash your commits? :+1:

**Comment by gmunkhbaatarmn** [comment]
Commits squashed into one commit.

**Comment by alebcay** [comment]
Looks good, thanks! :+1:

**Pull request merged**